### PR TITLE
Frequently used reacji bar component and stories

### PR DESCRIPTION
**Pull request by buoyad**
reacji-bar

Might change a bit depending on what the emoji picker ends up attaching to. r? @keybase/react-hackers 

- [ ] Storyshots before merge

### CODE REVIEW

**Comment by buoyad** [comment]
@keybase/react-hackers bump

**Comment by buoyad** [comment]
I'm changing this not to use `FloatingBox`

**Comment by buoyad** [comment]
OK, now this uses `className`, that makes more sense for integrating with the chat thread view. @keybase/react-hackers PTAL

**Comment by buoyad** [comment]
@keybase/react-hackers bump

File: shared/chat/conversation/messages/react-button/emoji-row/index.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../../../../../common-adapters'
+import * as Styles from '../../../../../styles'
+
+type Props = {
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/react-button/emoji-row/index.js]: exact

File: shared/chat/conversation/messages/react-button/emoji-row/index.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    </Kb.Box2>
+  </HoverBox>
+)
+
+const styles = Styles.styleSheetCreate({
+  container: Styles.platformStyles({
+    isElectron: {
+      backgroundColor: Styles.globalColors.white,
+      borderColor: Styles.globalColors.black_10,
+      borderRadius: 4,
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/react-button/emoji-row/index.js]: Styles.borderRadius

File: shared/chat/conversation/messages/react-button/emoji-row/index.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      borderStyle: 'solid',
+      borderWidth: 1,
+    },
+  }),
+  emojiBox: {
+    ...Styles.globalStyles.flexBoxRow,
+    alignItems: 'center',
+    height: 16,
+    justifyContent: 'center',
+    width: 16,
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/react-button/emoji-row/index.js]: globalMargins.small

**Pull request merged**